daily-challenges/2024-09/20/hunghoang3110.spec.ts

```
@@ -0,0 +1,44 @@
+// Javascript
+function findLargestNumber(array) {
+
+    let maxNumber = Math.max(...array);
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'maxNumber' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)